internal/schema/decoder.go

```diff
@@ -160,7 +160,17 @@ func isEmptyFields(fields []fieldWithPrefix, src map[string][]string) bool {
 				// issue references:
 				// https://github.com/gofiber/fiber/issues/1414
 				// https://github.com/gorilla/schema/issues/176
-				if !isEmpty(f.typ, src[key]) && key == path {
+				nested := strings.Contains(key, ".")
```

Comment:
use strings.IndexByte() ... it should be faster

Reply:
done

Reply:
thx